server/src/main/java/org/candlepin/controller/CandlepinPoolManager.java

```diff
@@ -1586,10 +1567,10 @@ else if (!poolQuantities.keySet().equals(entitlements.keySet())) {
         }
 
         handler.handlePostEntitlement(this, consumer, entitlements);
-        handler.handleSelfCertificates(consumer, poolQuantities, entitlements, generateUeberCert);
+        handler.handleSelfCertificates(consumer, poolQuantities, entitlements);
```

Comment:
With this no longer having the option to generate ueber certs, is there ever a point where we're at risk of not updating a ueber cert when we need to here? Or are they so far out of band now that it's not a concern?